qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -577,6 +585,67 @@ def test_snap_schedule_all_periods(self):
         self.mount_a.run_shell(['rmdir', test_dir])
 
 
+class TestSnapSchedulesSubvolAndGroupArguments(TestSnapSchedulesHelper):
+    def test_snap_schedule_subvol_and_group_arguments(self):
```

Comment:
I think this can be split into multiple tests.

Reply:
done ... split into multiple tests